pkg/config/sync.go

```diff
@@ -242,6 +242,11 @@ func mergeConfig(from, to *controller.OperatorConfiguration) {
 		if from.Workspace.PodSecurityContext != nil {
 			to.Workspace.PodSecurityContext = from.Workspace.PodSecurityContext
 		}
+		for storageClassName, PVCSize := range from.Workspace.StorageClassSize {
+			if PVCSize != defaultConfig.Workspace.StorageClassSize[storageClassName] {
+				to.Workspace.StorageClassSize[storageClassName] = PVCSize
```

Comment:
`to.Workspace.StorageClassSize` may be `nil` here, resulting in a panic. The merge code for storageClassSize should look something like 
```go
		if from.Workspace.StorageClassSize != nil {
			if to.Workspace.StorageClassSize == nil {
				to.Workspace.StorageClassSize = map[string]resource.Quantity{}
			}
			for storageClassName, PVCSize := range from.Workspace.StorageClassSize {
				to.Workspace.StorageClassSize[storageClassName] = PVCSize
			}
		}
```